src/test/javascript/spec/component/programming-exercise/programming-exercise-update.component.spec.ts

```diff
@@ -321,6 +323,20 @@ describe('ProgrammingExercise Management Update Component', () => {
             expect(comp.staticCodeAnalysisAllowed).toBe(true);
             expect(comp.packageNamePattern).toBe(comp.packageNamePatternForJavaKotlin);
         }));
+
+        it('Should deactivate SCA for C (FACT)', fakeAsync(() => {
+            // WHEN
+            fixture.detectChanges();
+            tick();
+            comp.onProgrammingLanguageChange(ProgrammingLanguage.C);
+            comp.onProjectTypeChange(ProjectType.FACT);
+
+            // THEN
+            expect(comp.selectedProgrammingLanguage).toEqual(ProgrammingLanguage.C);
```

Comment:
Can we use ``toBe`` in all 4 assertions here?

Reply:
Have now been replaced, thanks 👍